src/System.IdentityModel.Tokens/AsymmetricSignatureProvider.cs

```diff
@@ -158,21 +152,15 @@ protected virtual HashAlgorithmName GetHashAlgorithmName(string algorithm)
 
             switch (algorithm)
             {
-                case SecurityAlgorithms.SHA256:
```

Comment:
Given we don't know how the HashAlgorithmName is going to be used, why should we choose to claim we don't support it?